dev-libs/bcunit/bcunit-3.0.2.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A unit testing framework for C"
+DESCRIPTION="A lightweight system for writing, administering, and running unit \
+tests in C."
+HOMEPAGE="https://github.com/BelledonneCommunications/bcunit"
+COPYRIGHT="2017 BCUnit Authors"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/BelledonneCommunications/bcunit/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="eb7090aca5006aa0cea425722cb3fa1a7139185a7d9d1734d57844f69084713f"
+PATCHES="bcunit-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="1.0.1"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	bcunit$secondaryArchSuffix = $portVersion
+	lib:libbcunit$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	bcunit${secondaryArchSuffix}_devel = $portVersion
+	devel:libbcunit$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	bcunit$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize
```

Comment:
```diff
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
```
(Yes, I know this would not be required if we did not bundle libtoolize with libltdl...)

Everything else looks OK.